client/src/components/App.js

```diff
@@ -174,6 +208,8 @@ const mapDispatchToProps = dispatch => ({
   logout: () => dispatch(logout()),
   getPlaylist: () => dispatch(getPlaylist()),
   togglePlaying: () => dispatch(togglePlaying()),
+  setPlayingToTrue: () => dispatch(setPlayingToTrue()),
+  setPlayingToFalse: () => dispatch(setPlayingToFalse()),
```

Comment:
Instead of using two separate action creators for updating same state prop (`setPlayingToTrue` and `setPlayingToFalse`) you should create a single action creator, say `setPlayingState(isPlaying)`.